File: libraries/hdm/functions/types/response/DeleteObjectResponseDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.response.DeleteObjectResponseDTO') //< public
.satisfies('hdm.functions.types.response.ResponseDTO')
.satisfies('java.io.Serializable')
.protos({
    status:0, //< private int
    //> public int getStatus()
    getStatus:function(){
        return this.status;
    },
    //> public void setStatus(int status)
    setStatus:function(status){
        this.status=status;
    }
})
.endType();

File: libraries/hdm/functions/types/response/DeleteObjectResponseDTO.java
package hdm.functions.types.response;

import java.io.Serializable;

public class DeleteObjectResponseDTO
  implements ResponseDTO, Serializable
{
  private int status;

  public int getStatus()
  {
    return this.status;
  }

  public void setStatus(int status)
  {
    this.status = status;
  }
}